RNA-Seq/tutorials/ref_based.md

```diff
@@ -42,10 +42,9 @@ In this tutorial, we will analyze the data with:
 
 ## Data upload
 
-The data is available at NCBI Gene Expression Omnibus (GEO) under accession number [GSE18508](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE18508).
-Here, we have 7 samples:
-- 3 treated samples with Pasilla (PS) gene depletion (called "pasilla-depleted")
-- 4 untreated samples (called "wt")
+We will look at 7 samples:
```

Comment:
It might be better to keep the link to GEO, but then it should be known how to get the data from there. Is it extracted from the GSM461177 gff or wig file?